src/api/challenge.ts

```diff
@@ -239,7 +239,14 @@ export class ChallengeAPIClient {
     const challengeId = (await this._apiCreateChallenge(payoutId, limit, chain, isVerifiedJoin))
       .id;
     const challenge = await this._apiGetChallenge(challengeId);
-    return { challengeId, blockchainAddress: challenge.blockchainAddress! };
+    return {
+      challengeId,
```

Comment:
it's a bit odd to me that we return challengeId instead of id, like everywhere else.
I propose we keep challengeId for legacy and not break anything, but also return id